data/groupedbaryw.json

```diff
@@ -0,0 +1,15 @@
+[{"id":0,"type":132,"xval":"Aug 20","yval":74.928,"name":"Total Average","breakdown":"1634","breakdowntype":"YW","parentrefid":163,"parentreftype":135,"zval":1300},
```

Comment:
json file with data that have "YW" breakdowntype. need to verify with Jose the xvals since there are some mix of dates and strings